setup.cfg

```diff
@@ -49,6 +49,21 @@ minversion = 3.0
 norecursedirs = build docs/_build
 doctest_plus = enabled
 addopts = -p no:warnings
+markers =
+	slow: takes more than 12.5 minutes. Deselect with '-m "not slow"'
+	veryslow: takes more than 1 hour. Deselect with '-m "not veryslow"'
+	network: requires internet connection. Deselect with '-m "not network"'
+
+# The following are provided if using the "pytest-optional-tests" package, which
+#	can de-select by default instead of selecting by default. If using it,
+#	comment out the above 'markers' section. To run optional tests, run pytest
+#	with '--run-optional-tests=comma,separated,list'.
+#
+# optional_tests:
+#   slow: tests that require >15 minutes to execute
+#   veryslow: tests that require >1 hour to execute
+#   network: tests taht require network access
+  
```

Comment:
I don't think we will be using `pytest-optional-tests` we have this info in the commit history so it can be removed from the final draft

Reply:
Changes made and committed.